html/about.html

```diff
@@ -0,0 +1,144 @@
+              <path fill-rule="evenodd" d="M2 13.5V7h1v6.5a.5.5 0 0 0 .5.5h9a.5.5 0 0 0 .5-.5V7h1v6.5a1.5 1.5 0 0 1-1.5 1.5h-9A1.5 1.5 0 0 1 2 13.5zm11-11V6l-2-2V2.5a.5.5 0 0 1 .5-.5h1a.5.5 0 0 1 .5.5z"/>
+              <path fill-rule="evenodd" d="M7.293 1.5a1 1 0 0 1 1.414 0l6.647 6.646a.5.5 0 0 1-.708.708L8 2.207 1.354 8.854a.5.5 0 1 1-.708-.708L7.293 1.5z"/>
+            </svg>
+          </a>
+           > <a href="#">About</a>
+        </div>
+      </div>
+      <section id="about-event" class="container-fullwidth">
+        <span class="pre-title mobile-visible">"Hello! Sharing World"</span>
+        <h1 class="title">Creative Commons Global Summit 2015</h1>
+        <hr class="hr desktop-visible">
+        <p class="introduction">
+          The Global Summit brings together the community of experts, 
+          academics, and activists who comprise the Creative Commons 
+          affiliate network in a different country every two years.
+          <br>
+          This year, we hope to expand out invitation list including 
+          organizations and individuals<span class="mobile-visible">.</span> 
+          <span class="desktop-visible">
+            who want to work with us on shared projects that advance the cause of the Commons, 
+            free culture and open knowledge.
+            <br>
+            So if you're active and engaged in the worlds of open ocntent and knowledge - free 
+            software advocates, Wikipedians, Open Knowledge, galleries, libraries, museums, 
+            archives, governments and foundations, lawyers and activits - we hope you consider 
+            joining us this year to build a stronger, more vibrant commons together.
+          </span>
+        </p>
+        <article id="contact">
+          <span>
+            Please contact us via email for any further questions
+            <br>
+            about CC Global Summit 2015!
+          </span>
+          <br>
+          <a href="#">ccsummit2015@cckorea.org</a>
+        </article>
+      </section>
+    </main>
+    <section id="about-logo" class="section">
+      <div class="section-context">
+        <h2>
+          Creative Commons 
+          <br>
+          Global Summit 2015 Logo
+        </h2>
+        <hr class="hr">
+        <span>
+          The logo of CC Global Summit 2015 was decided through the logo 
+          competition from June 8th to July 7th.
+        </span>
+        <span class="desktop-visible">
+          After online public voting and Selection Comitee's judging process, 
+          the logo from Naresh Agrawal won the prize.
+          <br>
+          After consultation with Mr. Agrawal we optimized the prize-winning-
+          logo for this year's summit as following:
+        </span>
+      </div>
+      <img src="../assets/images/about_logo.png" alt="CC Global Summit 2015 Logo">
+    </section>
+    <hr class="sections-hr">
+    <section class="section" id="about-summits">

```

Comment:
- [ ] Well done creating the bottom section, let's make it even better by aligning the elements to the center in the desktop version with correct margin and padding:

Template:
![Captura de pantalla 2022-04-14 a la(s) 8 10 33 p  m](https://user-images.githubusercontent.com/83717285/163502556-d0d8d50b-fd9e-49f4-8d2f-432fd312769c.png)

Your design:
![Captura de pantalla 2022-04-14 a la(s) 8 08 15 p  m](https://user-images.githubusercontent.com/83717285/163502583-a6a17a76-631e-42d4-b5fc-2d336307e967.png)